lib/assessment.js

```diff
@@ -13,14 +21,23 @@ var min = function min() {
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
-
+  var total = arguments[0];
+  for (var i = 0; i < arguments.length; i++) {
+    total += arguments[i];
+  };
+  return total;
 };
 
 // Question 3
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
 var newArray = function newArray(length, defaultValue) {
+var array = [];
+  for (var i = 0; i < length; i++) {
+//I am unsure how to set all the elements to the defaultValue.
```

Comment:
Assign the default value to the array at index `i`. This way, it will assign it to a different index in each loop iteration.